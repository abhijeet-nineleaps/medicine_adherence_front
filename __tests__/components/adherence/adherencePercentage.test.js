import Enzyme, { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { Provider } from "react-redux";
import React from "react";
import configureStore from "redux-mock-store";
import toJson from "enzyme-to-json";
import AdherencePercentage from "../../../src/components/adherence/adherencePercentage";

Enzyme.configure({ adapter: new Adapter() });
const mockStore = configureStore([]);

describe("test adherence per", () => {
  let store;
  store = mockStore({
    field: {
      name: "abc",
      value: "San",
    },
  });
  it("test category", () => {
    const wrapper = shallow(
      <Provider store={store}>
        <AdherencePercentage />
      </Provider>
    );

    expect.assertions(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});