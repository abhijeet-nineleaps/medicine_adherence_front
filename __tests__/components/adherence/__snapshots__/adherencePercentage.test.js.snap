// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test adherence per test category 1`] = `<Component />`;
